tests/db/test_bigchain_api.py

```diff
@@ -978,41 +978,51 @@ def test_get_owned_ids_calls_get_outputs_filtered():
     assert res == gof()
 
 
+@pytest.mark.tendermint
 def test_get_outputs_filtered_only_unspent():
     from bigchaindb.common.transaction import TransactionLink
-    from bigchaindb.core import Bigchain
-    with patch('bigchaindb.fastquery.FastQuery.get_outputs_by_public_key') as get_outputs:
+    from bigchaindb.tendermint.lib import BigchainDB
+
+    go = 'bigchaindb.tendermint.fastquery.FastQuery.get_outputs_by_public_key'
```

Comment:
:+1:, but not entirely sure why its named `go`.

Reply:
A short of **g**et_**o**utputs :)

Reply:
Yes , figured it out, thats why updated. 👍